testpilot/frontend/static-src/app/views/experiment-tour-dialog-view.js

```diff
@@ -30,7 +30,7 @@ export default BaseView.extend({
         <div class="tour-actions">
           <div class="tour-back"></div>
           <div class="tour-next"></div>
-          <button data-hook="cancel-modal-done" class="button default large no-display" data-l10n-id="tourDoneButton">Done</button>
+          <button data-hook="complete-modal-done" class="button default large no-display" data-l10n-id="tourDoneButton">Done</button>
```

Comment:
Hmm, looks like I may need to add another `expectedStepStates` here. Will work on tomorrow.